packages/react/src/Form/MultiStep/FormStepHeader.tsx

```diff
@@ -0,0 +1,110 @@
+import { CheckCircle } from '@obosbbl/grunnmuren-icons';
+import { ReactNode, useMemo } from 'react';
+import { FormStatus } from './FormStep';
+import classNames from 'clsx';
+
+export interface FormStepHeaderProps {
+  className?: string;
+  collapsed: boolean;
+  step: number;
+  onClick: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void;
+  children: ReactNode;
+  formStatus: FormStatus;
+}
+
+export const FormStepHeader = (props: FormStepHeaderProps) => {
+  const { className, collapsed, step, onClick, children, formStatus } = props;
+
+  return (
+    <button
+      type="button"
+      aria-label={collapsed ? `Åpne steg ${step}` : `Steg ${step} er åpent`}
+      onClick={onClick}
+      disabled={collapsed && formStatus !== 'completed'}
```

Comment:
her må vi bruke `aria-disabled` i stedet for `disabled`. Ellers blir knappen/headingen helt usynlig for skjermlesere. Hvis du tester voiceover får du ikke tabbet til den at all. Med aria-disabled er den fortsatt fokuserbar.